Index: lede/package/base-files/files/etc/banner
===================================================================
--- lede.orig/package/base-files/files/etc/banner	2016-06-02 02:43:57.601381806 +0200
+++ lede/package/base-files/files/etc/banner	2016-06-02 02:46:17.928201514 +0200
@@ -1,10 +1,13 @@
-     _________
-    /        /\      _    ___ ___  ___
-   /  LE    /  \    | |  | __|   \| __|
-  /    DE  /    \   | |__| _|| |) | _|
- /________/  LE  \  |____|___|___/|___|                      lede-project.org
- \        \   DE /
-  \    LE  \    /  -----------------------------------------------------------
-   \  DE    \  /    %N (%C, %R)
-    \________\/    -----------------------------------------------------------
+ ////\\\\
+ ////\\\\ You are missing a nice banner and logo
+ ////\\\\ in
+ ////\\\\ your
+ ////\\\\ firmware
+ ////\\\\ ....
+
+ Firmware Builder (%N %C rev %R)
+ %P - %S
+ 
+ https://github.com/lynxis/firmware-builder
+ -----------------------------------------------------
 
